app/ui/src/app/dashboard/dashboard_metrics/dashboard-metrics.component.ts

```diff
@@ -11,7 +11,8 @@ import {
 @Component({
   selector: 'syndesis-dashboard-metrics',
   templateUrl: './dashboard-metrics.component.html',
-  styleUrls: ['./dashboard-metrics.component.scss']
+  styleUrls: ['../dashboard.component.scss',
+    './dashboard-metrics.component.scss']
```

Comment:
In the future I'd suggest to split each path string per line to enhance readability:

````
styleUrls: [
  '../dashboard.component.scss',
  './dashboard-metrics.component.scss'
]